apollo-runtime/src/main/java/com/apollographql/apollo/subscription/AppSyncOperationMessageSerializer.kt

```diff
@@ -0,0 +1,81 @@
+package com.apollographql.apollo.subscription
+
+import com.apollographql.apollo.api.internal.json.JsonWriter
+import com.apollographql.apollo.api.internal.json.Utils
+import com.apollographql.apollo.api.internal.json.writeObject
+import com.apollographql.apollo.subscription.ApolloOperationMessageSerializer.writePayloadContentsTo
+import okhttp3.HttpUrl
+import okio.Buffer
+import okio.BufferedSink
+import java.util.Base64
+
+class AppSyncOperationMessageSerializer(
+    private val authorization: Map<String, Any?>
+) : OperationMessageSerializer by ApolloOperationMessageSerializer {
```

Comment:
Is the implementation by delegation needed here ? I think the same result could be done by actually implementing the `readServerMessage` method

```
  override fun readServerMessage(source: BufferedSource) = ApolloOperationMessageSerializer.readServerMessage(source)
 
```

It's a few more bytes but avoid using the "implement by delegate" pattern which isn't used a lot in the codebase and it might produce a bit less bytecode too.